packages/service-library/src/servicelib/fastapi/rabbitmq.py

```diff
@@ -42,7 +42,7 @@ async def reconnect(app: FastAPI):
 def setup_rabbit(
     app: FastAPI,
     *,
-    settings: RabbitSettings,
+    settings: RabbitSettings | None,
```

Comment:
Question: do we need it optional? It is kind of strange to setup rabbit without having any settings

Reply:
Good point, I corrected this.